plugins/core/src/test/kotlin/de/fayard/refreshVersions/core/versions/VersionsPropertiesModelTest.kt

```diff
@@ -0,0 +1,83 @@
+package de.fayard.refreshVersions.core.versions
+
+import de.fayard.refreshVersions.core.internal.versions.VersionsPropertiesModel
+import de.fayard.refreshVersions.core.internal.versions.readFromText
+import de.fayard.refreshVersions.core.internal.versions.toText
+import de.fayard.refreshVersions.core.testResources
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertTrue
+
+class VersionsPropertiesModelTest {
+
+    private val samplesDir = testResources.resolve("versions-properties-samples")
+
+    @Test
+    fun `test parsing old format`() {
+        samplesDir.resolve("old-format").listFiles()!!.filter {
+            it.isDirectory
+        }.also { check(it.isNotEmpty()) }.forEach {
+            val inputFile = it.resolve("input.properties")
+            val outputFile = it.resolve("output.properties")
+            val parsedModel = VersionsPropertiesModel.readFromText(inputFile.readText())
+            assertEquals(
```

Comment:
Why not using Kotest matchers?
They are very nice IMHO https://kotest.io/matchers/core/

Reply:
I was not familiar with them, that's why.

Now, I found them pretty straightforward, which is why you'll find them in my latest push.

Reply:
FYI, I reverted this commit because the output of kotest is different from junit5 backed kotlin.test, and as a result, failures are not parser by the IDE where it'd give you the nice diff view.